app/src/test/contrib/components/Responsibilities/DownloadCoverageComponentTests.tsx

```diff
@@ -58,6 +59,22 @@ describe("DownloadCoverageContentComponent", () => {
         expect(fetchNewToken.called).to.be.true;
     });
 
+    it("calling meth onDownloadClicked sets state prop downloadButtonEnabled to false for 5 seconds", function(done: DoneCallback) {
+        this.timeout(5020);
+        const props = makeProps({ coverageToken: "TOKEN" });
+        const component = shallow(<DownloadCoverageContentComponent {...props} />);
+        const instance = component.instance() as DownloadCoverageContentComponent;
+        expect(component.state().downloadButtonEnabled).to.be.equal(true)
+        instance.onDownloadClicked();
+        setTimeout(() => {
+            expect(component.state().downloadButtonEnabled).to.be.equal(false)
+        },70);
+        setTimeout(() => {
+            expect(component.state().downloadButtonEnabled).to.be.equal(true)
+            done();
+        },5010);
+    });
```

Comment:
Hmm, this makes this test take 5 seconds to run which is a bit long. Maybe parameterize the delay (via a React prop) so you can test this more rapidly?